codecov_cli/commands/labelanalysis.py

```diff
@@ -57,18 +57,16 @@
     "--dry-run",
     "dry_run",
     help=(
-        "Print list of tests to run AND tests skipped (and options that need to be added to the test runner) to stdout. "
-        + "Also prints the same information in JSON format. "
-        + "JSON will have keys 'ats_tests_to_run', 'ats_tests_to_skip' and 'runner_options'. "
-        + "List of tests to run is prefixed with ATS_TESTS_TO_RUN= "
-        + "List of tests to skip is prefixed with ATS_TESTS_TO_SKIP="
+        "Print list of tests to run AND tests skipped AND options that need to be added to the test runner to stdout. "
+        + "Choose format with --dry-run-format option. Default is JSON. "
```

Comment:
can we list the other options?

Reply:
It does list them in the helper text of `--dry-run-format`